docs/Training-ML-Agents.md

```diff
@@ -1,4 +1,93 @@
+
+    tensorboard --logdir=summaries
+
+And then opening the URL: [localhost:6006](http://localhost:6006).
+ 
+When training is finished, you can find the saved model in the `python/models` folder under the assigned run-id — in the cats example, the path to the model would be `python/models/cob_1/CatsOnBicycles_cob_1.bytes`.
+
+While this example used the default training hyperparameters, you can edit the [training_config.yaml file](#training-config-file) with a text editor to set different values. 
+
+### Command line training options
+
+In addition to passing the path of the Unity executable containing your training environment, you can set the following command line options when invoking `learn.py`:
+
+* `--curriculum=<file>` – Specify a curriculum json file for defining the lessons for curriculum training. See [Curriculum Training](Training-Curriculum-Learning.md) for more information.
+* `--keep-checkpoints=<n>` – Specify the maximum number of model checkpoints to keep. Checkpoints are saved after the number of steps specified by the `save-freq` option. Once the maximum number of checkpoints has been reached, the oldest checkpoint is deleted when saving a new checkpoint. Defaults to 5.
+* `--lesson=<n>` – Specify which lesson to start with when performing curriculum training. Defaults to 0.
+* `--load` – If set, the training code loads an already trained model to initialize the neural network before training. The learning code looks for the model in `python/models/<run-id>/` (which is also where it saves models at the end of training). When not set (the default), the neural network weights are randomly initialized and an existing model is not loaded.
+* `--run-id=<path>` – Specifies an identifier for each training run. This identifier is used to name the subdirectories in which the trained model and summary statistics are saved as well as the saved model itself. The default id is "ppo". If you use TensorBoard to view the training statistics, always set a unique run-id for each training run. (The statistics for all runs with the same id are combined as if they were produced by a the same session.)
+* `--save-freq=<n>` Specifies how often (in  steps) to save the model during training. Defaults to 50000.
+* `--seed=<n>` – Specifies a number to use as a seed for the random number generator used by the training code.
+* `--slow` – Specify this option to run the Unity environment at normal, game speed. The `--slow` mode uses the **Time Scale** and **Target Frame Rate** specified in the Academy's **Inference Configuration**. By default, training runs using the speeds specified in your Academy's **Training Configuration**. See [Academy Properties](Learning-Environment-Design-Academy.md#academy-properties).
```

Comment:
slow is not really to run at game speed, it is to run with the inference configuration rather than the training configuration. It is probably fine, maybe we do not need such level of detail here.